README.md

```diff
@@ -15,8 +15,19 @@ These endpoints are called during the agent subscription journey.
 
 The "refusal to deal with" and "manually assured" lists are maintained by HMRC helpdesk staff (stride users) in agent-helpdesk-frontend.
 
+## Running the tests
+
+    sbt test it/test
+
+## Running the tests with coverage
+
+    sbt clean coverageOn test it/test coverageReport
+
 ## Run the application locally
 
+    sm2 --start AGENT_ONBOARDING
```

Comment:
This can just be a single dash `-start` and same with the stop below. Also, shouldn't this be `sm2 -start AGENT_AUTHORISATION` as what is stated in the https://github.com/hmrc/agent-services-account-ui-tests?tab=readme-ov-file#services